ironmon_tracker/MoveData.lua

```diff
@@ -1,3756 +1,3768 @@
 	},
 	{
 		id = "262",
 		name = "Memento",
-		type = PokemonTypes.DARK,
-		power = NOPOWER,
+		type = PokemonData.Types.DARK,
+		power = Constants.NO_POWER,
 		pp = "10",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		summary = "User faints to lower the Attack and Special Attack stat of the target by two stages each.",
 	},
 	{
 		id = "263",
 		name = "Facade",
-		type = PokemonTypes.NORMAL,
+		type = PokemonData.Types.NORMAL,
 		power = "70",
 		pp = "20",
 		accuracy = "100",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		iscontact = true,
 		summary = "Power doubles if the user is poisoned, paralyzed, or burned.",
 	},
 	{
 		id = "264",
 		name = "Focus Punch",
-		type = PokemonTypes.FIGHTING,
+		type = PokemonData.Types.FIGHTING,
 		power = "150",
 		pp = "20",
 		accuracy = "100",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		iscontact = true,
 		priority = "-- 3",
 		summary = "This is a decreased priority move. This move will fail if the user is dealt direct damage first.",
 	},
 	{
 		id = "265",
 		name = "SmellingSalt",
-		type = PokemonTypes.NORMAL,
+		type = PokemonData.Types.NORMAL,
 		power = "60",
 		pp = "10",
 		accuracy = "100",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		iscontact = true,
 		summary = "Power doubles against a paralyzed target, but it will also cure the target of paralysis.",
 	},
 	{
 		id = "266",
 		name = "Follow Me",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "20",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		priority = "+ 3",
 		summary = "This is an increased priority move. The user redirects all targeted moves by enemy Pokemon to itself.",
 	},
 	{
 		id = "267",
 		name = "Nature Power",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "20",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		summary = "Sand: Earthquake, Building: Swift, Cave: Shadow Ball, Rock: Rock Slide, Tall grass: Stun Spore, Long grass: Razor Leaf, Pond: Bubblebeam, Sea: Surf, Underwater: Hydro Pump.",
 	},
 	{
 		id = "268",
 		name = "Charge",
-		type = PokemonTypes.ELECTRIC,
-		power = NOPOWER,
+		type = PokemonData.Types.ELECTRIC,
+		power = Constants.NO_POWER,
 		pp = "20",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		summary = "User charges up, doubling the damage dealt by the next move it uses but only for Electric moves.",
```

Comment:
could maybe reword into "doubling the damage dealt by the next electric-type move"